selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/projectexplorer/DownloadProjectTest.java

```diff
@@ -103,19 +104,27 @@ public void downloadAllProjectsTest() throws IOException {
     // given
     List<String> expectedPackageFileList =
         asList(
-            "TestProject1/.che/classpath",
+            ".che/user_che/appState",
+            "TestProject1/.classpath",
+            "TestProject1/.project",
+            "TestProject1/.settings/org.eclipse.jdt.apt.core.prefs",
+            "TestProject1/.settings/org.eclipse.jdt.core.prefs",
+            "TestProject1/.settings/org.eclipse.m2e.core.prefs",
             "TestProject1/README.md",
             "TestProject1/pom.xml",
             "TestProject1/src/main/java/org/eclipse/qa/examples/AppController.java",
             "TestProject1/src/main/webapp/WEB-INF/jsp/guess_num.jsp",
             "TestProject1/src/main/webapp/WEB-INF/spring-servlet.xml",
             "TestProject1/src/main/webapp/WEB-INF/web.xml",
             "TestProject1/src/main/webapp/index.jsp",
-            "TestProject2/.che/classpath",
-            "TestProject2/README.md",
+            "TestProject2/.classpath",
+            "TestProject2/.project",
+            "TestProject2/.settings/org.eclipse.jdt.apt.core.prefs",
+            "TestProject2/.settings/org.eclipse.jdt.core.prefs",
+            "TestProject2/.settings/org.eclipse.m2e.core.prefs",
             "TestProject2/pom.xml",
-            "TestProject2/src/main/java/org/eclipse/qa/examples/AppController.java",
-            "TestProject2/src/main/webapp/WEB-INF/jsp/guess_num.jsp",
+            "TestProject2/src/main/java.org.eclipse.qa.examples/AppController.java",
```

Comment:
This does not look like a file from "/projects/ProjectWithDifferentTypeOfFiles2". Also, shouldn't we have a folder structure here?